qa/tasks/ceph_manager.py

```diff
@@ -13,10 +13,9 @@
 import threading
 import traceback
 import os
-import six
 
 from io import BytesIO
```

Comment:
Nit: perhaps combine? `from io import BytesIO, StringIO`

Reply:
updated and repushed.